src/Components/Aspire.OracleManagedDataAccess/AspireOracleManagedDataAccessExtensions.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Data.Common;
+using Aspire;
+using Aspire.OracleManagedDataAccess;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Oracle.ManagedDataAccess.Client;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Extension methods for connecting Oracle database with Oracle.ManagedDataAccess client
+/// </summary>
+public static class AspireOracleManagedDataAccessExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:OracleManagedDataAccess";
+
+    /// <summary>
+    /// Registers <see cref="OracleConnection"/> for connecting Oracle database with Oracle.ManagedDataAccess client.
+    /// Configures health check, logging and telemetry for the Oracle.ManagedDataAccess client.
+    /// </summary>
+    /// <param name="builder">The <see cref="IHostApplicationBuilder" /> to read config from and add services to.</param>
+    /// <param name="connectionName">A name used to retrieve the connection string from the ConnectionStrings configuration section.</param>
+    /// <param name="configureSettings">An optional delegate that can be used for customizing options. It's invoked after the settings are read from the configuration.</param>
+    /// <remarks>Reads the configuration from "Aspire:Oracle" section.</remarks>
```

Comment:
Aspire:OracleManagedDataAccess

Reply:
Changed to Aspire:Oracle:ManagedDataAccess:Core as @eerhardt advised